scripts/test_repo.sh

```diff
@@ -12,6 +12,7 @@ for file in $(git --work-tree="$repo_path" --git-dir="$repo_path/.git" ls-files
 do
   full_path="$repo_path/$file"
   errors=$(ruby --disable=gems src/rubyfmt.rb "$full_path" 2>&1)
+  # shellcheck disable=SC2181
```

Comment:
I disabled this lint since I didn't come up with a way to run a command, and save its output.

@georgebrock might have a trick up his sleeve.

Reply:
I don't know another way of doing this, aside from the one I already used in the original script.